keps/sig-network/2004-topology-aware-subsetting/README.md

```diff
@@ -364,7 +366,7 @@ endpoints as full as possible.
    - `endpoints_available` pool is empty.
    - `endpoints_require` pool is empty.
 4. If `endpoints_available` pool is empty and `endpoints_require` pool is not
-   empty, downgrade to `Random` approach.
+   empty, downgrade to `Disabled` approach.
```

Comment:
[Comment to this whole subsection - not to this line].

How we're planning to reduce the churn on EndpointSlices?
Please correct me if I'm wrong, by my understanding is that this algorithm will be run each time when we will be computing slicing for a given service.

So let's say I have all my endpoints in zone A (say 30 endpoints). How you're going to achieve the situation that e.g. when adding 1 endpoint only one EndpointSlice will get updated, not potentially all 3 of them?

[This algorithm doesn't seem to take inco acount a current assignment...]